hack/make/build-deb

```diff
@@ -3,6 +3,8 @@ set -e
 
 # subshell so that we can export PATH and TZ without breaking other things
 (
+	# Keep list uptodate with what uses systemd >= 226 until everything does
+	export new_systemd=("ubuntu-xenial" "debian-stretch")
```

Comment:
Yes, that is a good idea. That way, we've got a list, and we don't have to change it to pick up the value for any new release. And if an old release is removed, we don't *have* to change the list then either. Testing this now.  Thanks @tianon!